worlds/earthbound/Rom.py

```diff
@@ -0,0 +1,853 @@
+            world.starting_money += money_item_table[world.multiworld.get_location("Poo - Starting Item", world.player).item.name]
+
+        # if item.name == "Poo" and world.multiworld.get_location("Poo - Starting Item", world.player).item.name == "Photograph":
+            # rom.write_bytes(0x17FEA9, bytearray([0x01]))
+
+        if item.name in ["Progressive Bat", "Progressive Fry Pan", "Progressive Gun", "Progressive Bracelet",
+                         "Progressive Other"]:
+            old_item_name = item.name
+            item.name = world.progressive_item_groups[item.name][world.start_prog_counts[item.name]]
+            if world.start_prog_counts[old_item_name] != len(world.progressive_item_groups[old_item_name]) - 1:
+                world.start_prog_counts[old_item_name] += 1
+
+        if item.name in item_id_table:
+            rom.write_bytes(0x375000 + starting_item_address, bytearray([item_id_table[item.name]]))
+            starting_item_address += 1
+        elif item.name in psi_item_table:
+            if item.name != "Progressive Poo PSI":
+                if item.name not in starting_psi_types:
+                    rom.write_bytes(0x17FC7C + starting_psi, bytearray([starting_psi_table[item.name]]))
+                    starting_psi_types.append(item.name)
+                    starting_psi += 1
+            else:
+                if starting_psi_types.count(item.name) < 2:
+                    rom.write_bytes(0x17FC7C + starting_psi, bytearray([starting_psi_table[item.name]]))
+                    starting_psi_types.append(item.name)
+                    starting_psi += 1
+        elif item.name in character_item_table and item.name != "Photograph":
+            if item.name not in starting_character_count:
+                rom.write_bytes(0x17FC8D + starting_char, bytearray([party_id_nums[item.name]]))
+                starting_character_count.append(item.name)
+                starting_char += 1
+        elif item.name in money_item_table:
+            world.starting_money += money_item_table[item]
+
+    if world.options.random_battle_backgrounds:
+        bpp2_bgs = [bg_id for bg_id, bpp in battle_bg_bpp.items() if bpp == 2]
+        bpp4_bgs = [bg_id for bg_id, bpp in battle_bg_bpp.items() if bpp == 4]
+        for i in range(483):
+            world.flipped_bg = world.random.randint(0, 100)
+            if i == 480:
+                drawn_background = struct.pack("H", 0x00E3)
+            else:
+                drawn_background = struct.pack("H", world.random.randint(0x01, 0x0146))  # clearly this isn't giygas
+
+            if battle_bg_bpp[struct.unpack("H", drawn_background)[0]] == 4:
+                drawn_background_2 = struct.pack("H",  0x0000)
+            else:
+                drawn_background_2 = struct.pack("H", world.random.choice(bpp2_bgs))
+            if world.flipped_bg > 33 or drawn_background not in bpp2_bgs:
```

Comment:
```suggestion
            if world.flipped_bg > 33 or drawn_background not in bpp2_bgs:
```
`drawn_background` is a bytes object; this can only be true if `world.flipped_bg > 33`.